system-x/services/hawtio/src/main/java/software/tnb/hawtio/client/openshift/OpenshiftHawtioClient.java

```diff
@@ -0,0 +1,60 @@
+package software.tnb.hawtio.client.openshift;
+
+import software.tnb.common.openshift.OpenshiftClient;
+import software.tnb.hawtio.client.HawtioClient;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import io.fabric8.kubernetes.client.http.HttpClient;
+import io.fabric8.kubernetes.client.http.HttpRequest;
+import io.fabric8.kubernetes.client.http.HttpResponse;
+
+public class OpenshiftHawtioClient implements HawtioClient {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OpenshiftHawtioClient.class);
+
+    private final HttpClient client;
+    private final String baseUrl;
+
+    public OpenshiftHawtioClient(String hawtioUrl) {
+        this.client = OpenshiftClient.get().authorization().getHttpClient();
```

Comment:
since you're using a route to access hawtio, wouldn't HTTPUtils be sufficient here as well?

Reply:
I receive 401/403 using unauthenticated client, so that I prefer to use the provided/configured fabric8 client